package/crossplane.yaml

```diff
@@ -23,11 +23,12 @@ metadata:
ODcgNDQuMzkgNi45NyA0NC4zOSA2Ljk3IDQzIDEuODcgNDMgMS44NyA0MC4wMiA5LjQ3IDQwLjAyIDkuNDcgMzguNjMgMC4zMyAzOC42MyAwLjMzIDQ5LjE4IDkuNzQgNDkuMTggOS43NCA0Ny44MiAxLjg3IDQ3LjgyIDEuODcgNDQuMzkiLz48cGF0aCBkPSJNMjk2LjA1LDM5MC41OWEyLjk0LDIuOTQsMCwwLDEtMy4xLDMuMTMsMi44NSwyLjg1LDAsMCwxLTIuOTMtMy4xM3YtNi4zMUgyODguNXY2LjM4YTQuMjQsNC4yNCwwLDAsMCw0LjQzLDQuNDVjMy40OSwwLDQuNjMtMi4zOCw0LjYzLTQuNDV2LTYuMzhoLTEuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMjUzLjk2IC0zNDUuNjEpIi8+PHJlY3QgeD0iNTIuMDkiIHk9IjM4LjYzIiB3aWR0aD0iMS41MSIgaGVpZ2h0PSIxMC41NSIvPjxwb2x5Z29uIHBvaW50cz0iNzEuNjggNDYuOSA2NC4yIDM4LjYzIDYyLjcgMzguNjMgNjIuNyA0OS4xOCA2NC4yMiA0OS4xOCA2NC4yMiA0MC44NSA3MS43MSA0OS4xOCA3My4xOSA0OS4xOCA3My4xOSAzOC42MyA3MS42OCAzOC42MyA3MS42OCA0Ni45Ii8+PHJlY3QgeD0iODIuMzciIHk9IjM4LjYzIiB3aWR0aD0iMS41MSIgaGVpZ2h0PSIxMC41NSIvPjxwb2x5Z29uIHBvaW50cz0iOTguNjEgNDMuNDYgMTAzIDM4LjYyIDEwMS4xMyAzOC42MiA5Ny42NCA0Mi40MSA5NC4yNSAzOC42MiA5Mi4yOCAzOC42MiA5Ni41MSA0My40NiA5MS4yMiA0OS4xOCA5My4xNSA0OS4xOCA5Ny41IDQ0LjQ4IDEwMS42NCA0OS4xOCAxMDMuNzQgNDkuMTggOTguNjEgNDMuNDYiLz48cGF0aCBkPSJNMjc1LjIsMzgzLjgyYTUuNTEsNS41MSwwLDAsMC01LjU2LDUuNzJ2MGE1LjQ0LDUuNDQsMCwwLDAsNS42LDUuNiw2LDYsMCwwLDAsMi40Ni0uNThsLjg2LDFoMS43NmwtMS41MS0xLjcyYTUuMzcsNS4zNywwLDAsMCwyLTQuNHYwYTUuNDcsNS40NywwLDAsMC01LjYyLTUuNm00LjA1LDUuN2E0LjEyLDQuMTIsMCwwLDEtMS4zNiwzLjI4bC0uODItLjk0aC0xLjc2bDEuNDQsMS42NWEzLjkzLDMuOTMsMCwwLDEtMS40OC4zMSw0LjA4LDQuMDgsMCwwLDEtNC00LjI4djBhNCw0LDAsMCwxLDMuOTUtNC4zMiw0LjEsNC4xLDAsMCwxLDQuMDgsNC4zWiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTI1My45NiAtMzQ1LjYxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTTMzMy43NywzNjUuNzRhMi42LDIuNiwwLDAsMCwyLjU4LTIuNmgwYTIuNTgsMi41OCwwLDEsMC01LjE2LDB2MGEyLjU3LDIuNTcsMCwwLDAsMi41OCwyLjU4bTAtLjI4YTIuMjcsMi4yNywwLDAsMS0yLjI4LTIuM3YwYTIuMjksMi4yOSwwLDEsMSw0LjU3LDBoMGEyLjMsMi4zLDAsMCwxLTIuMjksMi4zMm0tMS4wNi0xaC42NXYtLjg0aC40MWwuNTguODRoLjc5bC0uNjgtMWEuODMuODMsMCwwLDAsLjU5LS44M2MwLS42MS0uNDYtLjg5LTEuMDktLjg5aC0xLjI1Wm0uNjUtMS4zN3YtLjc1aC41NmMuMjgsMCwuNDYuMTQuNDYuMzdzLS4xNi4zOC0uNDYuMzhaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMjUzLjk2IC0zNDUuNjEpIi8+PHBhdGggY2xhc3M9ImNscy0yIiBkPSJNMjczLjcyLDQyOC4zN0gyNzZsMy42OSw1LjQ2LDMuNjktNS40NmgyLjI2djExLjU0SDI4My41di04LjI4bC0zLjgyLDUuNDVoLS4wN2wtMy44LTUuNDF2OC4yNGgtMi4wOVoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0yNTMuOTYgLTM0NS42MSkiLz48cGF0aCBjbGFzcz0iY2xzLTIiIGQ9Ik0yOTAuMTQsNDI4LjM3aDl2MS44MWgtNi44NnYzaDYuMDhWNDM1aC02LjA4djMuMDloN3YxLjgxaC05LjA4WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTI1My45NiAtMzQ1LjYxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTMwNi4wOCw0MzAuMjVoLTMuODZ2LTEuODhoOS44NXYxLjg4aC0zLjg2djkuNjZoLTIuMTNaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMjUzLjk2IC0zNDUuNjEpIi8+PHBhdGggY2xhc3M9ImNscy0yIiBkPSJNMzE4LjQzLDQyOC4yOGgybDUuMzMsMTEuNjNoLTIuMjVsLTEuMjMtMi43OGgtNS43M2wtMS4yNSwyLjc4SDMxMy4xWm0zLjA1LDcuMDUtMi4wOS00LjYyLTIuMDgsNC42MloiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0yNTMuOTYgLTM0NS42MSkiLz48cGF0aCBjbGFzcz0iY2xzLTIiIGQ9Ik0zMjkuMjEsNDI4LjM3aDIuMTN2OS43aDYuMzd2MS44NGgtOC41WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTI1My45NiAtMzQ1LjYxKSIvPjxyZWN0IGNsYXNzPSJjbHMtMiIgeT0iNjUuNTIiIHdpZHRoPSIxMDQuMDYiIGhlaWdodD0iMC41Ii8+PC9zdmc+
```

Comment:
Added an icon for the service: https://gist.github.com/displague/163c0fdf2ff15c52a58a3df4020c193a#file-em-stacked-color-svg (the square one)